csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -5,9 +5,9 @@
 
 # Common Security Advisory Framework Version 2.0
 
-## Committee Specification 01
+## Committee Specification Draft 02
```

Comment:
I intended to create a PR from some ephemeral branch to the editor branch but my fingers chose differently.

Personally, I do not think that this is no problem as the change set is very small and clear in scope. So, I hope the TC members see it the same way ... 🙈